packages/next/src/client/components/react-dev-overlay/_experimental/internal/icons/external-small.tsx

```diff
@@ -0,0 +1,18 @@
+export function ExternalSmall() {
+  return (
+    <svg
+      width="16"
```

Comment:
width/height should be prop, also probably remove `fill` also for other icons since it's also an prop can be ocntrolled.

definition: `ExternalIcon(props)`

usage: `<ExternalIcon width={16}>`